### PR TITLE
Fix issue with zoom level

### PR DESCRIPTION
**Pull request by fcsonline**
Pending tasks:
- [x] Remove the build commit

### CODE REVIEW

File: scrollinga.js [hunk 1]
```diff
@@ -127,7 +127,10 @@ class Scrollinga {
    * @return {Boolean}
    */
   isScrollDown() {
-    return (this.target.scrollTop + this.target.clientHeight === this.target.scrollHeight);
+    const client = Math.floor((this.target.clientHeight + this.target.scrollTop) / devicePixelRatio);
```
Comment (andresgutgon) [inline scrollinga.js]: Maybe better with `window.devicePixelRation`. Also not IE10 do not have support for it. Maybe a fallback
http://caniuse.com/#feat=devicepixelratio

**Comment by fcsonline** [comment]
Better @andresgutgon ?

File: dist/scrollinga.js [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+	  return Scrollinga;
+	})();
+
+	exports['default'] = Scrollinga;
+	module.exports = exports['default'];
+
+/***/ }
+/******/ ])
+});
+;
```
Comment (andresgutgon) [inline dist/scrollinga.js]: ~~Why are you including the build in the repo? And if it's for distribution maybe apply some kind of uglify?~~

I :eyes:  the TODO

File: scrollinga.js [hunk 3]
```diff
@@ -127,7 +127,11 @@ class Scrollinga {
    * @return {Boolean}
    */
   isScrollDown() {
-    return (this.target.scrollTop + this.target.clientHeight === this.target.scrollHeight);
+    const pixelRatio = window.devicePixelRatio || 1;
+    const client = Math.floor((this.target.clientHeight + this.target.scrollTop) / pixelRatio);
+    const scroll = Math.floor(this.target.scrollHeight / pixelRatio);
+
+    return client === scroll;
```
Comment (andresgutgon) [inline scrollinga.js]: Maybe better names like `clientHeight` and `scrollHeight` to map original not pixelated variables

**Comment by andresgutgon** [comment]
:postal_horn: :+1:

File: dist/scrollinga.js (hunk 2 above)
Comment (fcsonline) [inline dist/scrollinga.js]: It's just a way to pass QA in Teambox frontend. I'll remove it before publish the new scrollinga version.

File: scrollinga.js (hunk 3 above)
Comment (fcsonline) [inline scrollinga.js]: ok

**Comment by Morantron** [comment]
go go go

**Pull request merged**